src/main/java/ch/uzh/ifi/hase/soprafs22/service/InGameWebsocketService.java

```diff
@@ -107,6 +111,39 @@ public Move verifyMove(Game game, Move move, String username){
         // If everything went well, return the move that was made
         return move;
     }
+    
+    public void afterMove(Move move, String uuid) {
```

Comment:
Always try to think of: "what does this function do?" and name it after that. I would suggest naming it differently.
```suggestion
    public void notifyPlayersAfterMove(Move move, String uuid) {
```

Reply:
True, renamed it in latest commit